feat: change config fields name to normalize

bolinocroustibat opened the pull request.

dtrckd reviewed (commented):
Ok, just, I find the previous values more explicit for the following variables
- ALBERT_API_URL
- ALBERT_API_TOKEN

Because there are many URLs, and for the token, it's more in line with what already exists. For example, the standards are "MISTRAL_API_TOKEN" OPENAI_API_TOKEN".

bolinocroustibat commented:
> Ok, just, I find the previous values more explicit for the following variables
> 
> * ALBERT_API_URL
> * ALBERT_API_TOKEN
> 
> Because there are many URLs, and for the token, it's more in line with what already exists. For example, the standards are "MISTRAL_API_TOKEN" OPENAI_API_TOKEN".

Agreed and changed.
Also changed the order of env vars so it's more consistent in all files.

dtrckd commented on app/pyalbert_utils.py:
```diff
@@ -37,8 +37,8 @@ def new_chat(config: Config) -> int:
+    api_model = config.albert_model_name
+    api_mode = config.albert_mode
     api_url = config.albert_api_url
     with_history = config.with_history
```
Didn't you change some variable names !?

bolinocroustibat replied:
Well spotted, fixed
